src/swatiShinde30/assignment3/Question11.java

```diff
@@ -0,0 +1,29 @@
+package swatiShinde30.assignment3;
+
+import java.util.Scanner;
+
+/*
+ * WAP to find number is palindrome or not ? 
+ */
+public class Question11 {
+	static void palindrom(String str) {
+		String str2 = "";
+		for (int i = str.length() - 1; i >= 0; i--) {
+			char ch = str.charAt(i);
```

Comment:
Everytime this variable created .. you should declare this variable above for loop